### ISSUE TITLE
Feature Request - Add a new CSV export option to Identifier Magicker tool

### ISSUE THREAD

**Issue by mkoo** [ArctosDB/arctos#7002]
**Problem is it solving**
Streamline identifier clean-up and bulk management. Specifically for filling in missing Issued_by for already existing identifiers

**Describe the solution:**
The simplest might be to add another export CSV type or Option to this tool "Identifier Magicker Thingee"(https://arctos.database.museum/tools/identifier_loader_down.cfm) which can be reviewed and seamlessly added to the tool Identifier IssuedBy Bulkloader (https://arctos.database.museum/loaders/bulk_identifier_issuedby.cfm).
The new export would have the following fields mainly from the results table
![Firefox_Screenshot_2023-11-21T20-06-05 225Z](https://github.com/ArctosDB/arctos/assets/2523089/db75ae27-b407-4172-9ffc-e4d293d4f1b8)

And the user would be able to fill in the field Issued_by (which needs to be Null to work)

Then preview and load like the other two Options

**Current pain it would solve/ alternatives**
Right now I have to merge two CSVs from two different places in order to get the CSV for the Identifier IssuedBy Bulkloader (do-able but unnecessary)

**Comment by dustymc** [comment]
I'm going to attempt to merge the magicker into the viewer/downloader, which should simplify things for everyone.

The complication is that the magicker returns one row per guid, while the downloader returns one row per identifier and allows filtering. Currently after a search finding 10 records one tool will return 10 records, and the other will return somewhere between zero (records have no identifiers, or all identifiers have been filtered out) and lots (lots of records may have lots of identifiers). It's not yet completely clear to me what that might look like in a merged tool, but I think it'll occasionally involve the tool returning more rows than the summary table contains (for records without identifiers, and maybe something about filters). I think the benefits of having all identifier-stuff in one place greatly outweigh the costs of any potential confusion, but someone please let me know NOW if I'm not seeing that correctly.
